client/styles/justgrid.css

```diff
@@ -1 +1,20 @@
-.row:after,.row:before{clear:both;content:"";display:table}.col,.row{box-sizing:border-box;width:100%}.col{float:left}.col.one{width:8.333%}.col.two{width:16.666%}.col.three{width:25%}.col.four{width:33.333%}.col.five{width:41.666%}.col.six{width:50%}.col.seven{width:58.333%}.col.eight{width:66.666%}.col.nine{width:75%}.col.ten{width:83.333%}.col.eleven{width:91.666%}@media(max-width:600px){.row:not(.static)>.col{width:100%}}
\ No newline at end of file
+/*
```

Comment:
Can/should we remove this commented-out code? Or should we consider revising the JobPaginator to use the MUI grid so we don't need this file at all? (I'm pretty sure this is only being used in JobPaginator.tsx, though maybe there are some other references I'm missing.